lib/blocks.php

```diff
@@ -512,3 +502,17 @@ function gutenberg_experimental_build_css_typography( $attributes, $supports ) {
 	return $typography;
 }
 
+/**
+ * Checks whether or not the given array has the given chain of nested keys.
+ * ex) $attributes['style']['color']['font']
+ *
+ * @param  array  $array array to check for keys.
+ * @param  string $key1 1st level key to check.
+ * @param  string $key2 2nd level key to check.
+ * @param  string $key3 3rd level key to check.
+ */
+function has_triple_nested_key( $array, $key1, $key2, $key3 ) {
```

Comment:
I think this function should be prepended with `gutenberg_experimental`, or at least `gutenberg` since we are not using namespaces here, and it could potentially clash with other code.

Reply:
Is `null` a valid value here that needs to be disambiguated? If not, you can do it in one call. 

```
isset( $array[ $key1 ][ $key2 ][ $key3 ] )
```

PHP won't error even if any of the earlier keys aren't defined.

Reply:
> Is null a valid value here that needs to be disambiguated? If not, you can do it in one call.

Oh, I think that will work well!  I don't think it should be set as `null`, but if it was we wouldn't want to apply the style anyways.

> PHP won't error even if any of the earlier keys aren't defined.

That is fantastic, thanks 😄